internal/docstore/firedocstore/fs.go

```diff
@@ -48,18 +63,28 @@ type collection struct {
 // firedocstore requires that a single string field, nameField, be designated the
 // primary key. Its values must be unique over all documents in the collection, and
 // the primary key must be provided to retrieve a document.
-func OpenCollection(client *vkit.Client, projectID, collPath, nameField string) *docstore.Collection {
-	return docstore.NewCollection(newCollection(client, projectID, collPath, nameField))
+func OpenCollection(client *vkit.Client, projectID, collPath string, opts Options) (*docstore.Collection, error) {
```

Comment:
Per your comment on the issue, I think this should be `*Options`, not `Options`. I suspect you made a non-pointer because it contains required options, but see above for an alternative.

Reply:
Obsolete.